apps/designer/app/shared/share-project/share-project.tsx

```diff
@@ -1,28 +1,284 @@
-import { Button, Flex, TextField } from "@webstudio-is/design-system";
+import {
+  Box,
+  Button,
+  css,
+  Flex,
+  Label,
+  DeprecatedPopover,
+  DeprecatedPopoverContent,
+  DeprecatedPopoverTrigger,
```

Comment:
Why here legacy components?

Reply:
we just deprecated them while this PR was still not ready, feel free to merge and update separately if you want